src/rules/max-line-length/README.md

```diff
@@ -74,3 +74,37 @@ a { color: pink; background: orange; }
  */
 a { color: pink; /* this comment is also long but not on its own line */ }
 ```
+
+### `ignore: ["comments"]`
+
+Only enforce the line-length limit for lines that are not commented out.
+
+This also applies to comments that are stuck in between other stuff.
```

Comment:
I see you've derived this from the `ignore: ["non-comments"]`([src](https://github.com/stylelint/stylelint/blob/master/src/rules/max-line-length/README.md#ignore-non-comments) option:
> _"This does not apply to comments that are stuck in between other stuff, only to lines that begin at the beginning or in the middle of a comment."_

Maybe something along the lines of:
> "This also applies to _all_ comments, including comments in between code on the same line."